docs/src/running-validator/vote-accounts.md

```diff
@@ -0,0 +1,115 @@
+---
+title: Vote Account Management
+---
+
+This page describes how to set up an on-chain _vote account_.   Creating a vote
+account is needed if you plan to run a validator node on Solana.
+
+All settings for a vote account can be configured at the time the account is
+created, or they can be set after the account is created while the corresponding
+validator is running.
+
+### Vote Account Address
+
+A vote account is created at an address that is either the public key of a
+keypair file, or at a derived address from the authorized voter's public key.
+
+The address of a vote account does not need to have a stored corresponding
+private key, because the address (although it may be a valid public key) is never
+needed to sign any transactions, but is just used to look up the account
+information.
+
+When someone wants to [delegate tokens in a stake account](../staking.md),
+the delegation command is pointed at the vote account address of the validator
+to whom the token-holder wants to delegate.
+
+### Validator Identity
+
+The _validator identity_ is a system account that pays for all the vote transaction
+fees submitted by the vote account.  Because the vote account is expected to vote
+on most valid blocks it receives, the validator identity account is frequently
+(as often as once per slot, or roughly every 400ms) signing transactions and
+paying fees.  For this reason the validator identity account must be stored as
+a "hot wallet" as a keypair file on the same system the validator process is
+running.
+
+Because a hot wallet is generally less secure than an offline or "cold" wallet,
+we only recommend storing enough SOL on the Identity keypair to cover voting fees
+for some amount of time (perhaps a few weeks or months) and periodically topping
+off your validator identity account from a more secure wallet.
+
+The validator identity is required to be provided when a vote account is created.
+The validator identity can also be changed after an account is created by using
+the `vote-update-validator` command.
+
+### Vote Authority
+
+The _vote authority_ keypair is used to sign each vote transaction the validator
+node wants to submit to the cluster.  This can be the same keypair used as the
+validator identity for the same vote account, or a different keypair.  Because
```

Comment:
```suggestion
node wants to submit to the cluster.  This doesn't necessarily have to be unique
from the validator identity, as you will see later in this document.  Because
```